embedded-graphics/src/geometry/size.rs

```diff
@@ -79,7 +79,13 @@ impl Size {
     ///
     /// let size = Size::new_equal(11);
     ///
-    /// assert_eq!(size, Size { width: 11, height: 11 });
+    /// assert_eq!(
+    ///     size,
+    ///     Size {
+    ///         width: 11,
+    ///         height: 11
+    ///     }
+    /// );
```

Comment:
Extra formatting from blanket `rustfmt` added by accident. Can't hurt to leave around even though it's irrelevant for the meat of this PR.